internal/database/postgres/migration/000005_edit-flight-test.up.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO flights (id, start_date, end_date, departure, destination, rank, price, total_tickets)
```

Comment:
is `postgrestest` missing? You can make a separate PR with it, if it's gonna be used in many PRs to avoid conflicts

Reply:
migration id conflicts with #67